generated/nidaqmx/system/system.py

```diff
@@ -329,29 +299,13 @@ def get_digital_power_up_states(self, device_name):
               Indicates the power up state set for the physical channel
               specified with the **physical_channel** output.
         """
-        states = []
-
         device = Device(device_name)
```

Comment:
This constructs `Device()` with the wrong interpreter. It should use `device = _DeviceAlternateConstructor(device_name, self._interpreter)`. 

There are a lot of cases like this. Some of them may be covered by AB#2366649.
```
PS D:\dev\nidaqmx-python> rg "Device\(" .\generated\nidaqmx\system
.\generated\nidaqmx\system\_collections\device_collection.py
60:            return Device(self.device_names[index])
62:            return [Device(name) for name in self.device_names[index]]
66:                return Device(device_names[0])
67:            return [Device(name) for name in device_names]
75:            yield Device(device_name)
88:            yield Device(device_name)

.\generated\nidaqmx\system\system.py
302:        device = Device(device_name)
372:        device = Device(device_name)
475:        device = Device(device_name)

.\generated\nidaqmx\system\device.py
53:        return f'Device(name={self._name})'
1445:        return [Device(v)
1789:        return Device(val.value.decode('ascii'))
1843:        return Device(val.value.decode('ascii'))
2043:        return [Device(v)
2077:        return Device(val.value.decode('ascii'))
2580:        device = Device("", grpc_options=grpc_options)

PS D:\dev\nidaqmx-python> rg "PhysicalChannel\(" .\generated\nidaqmx\system
.\generated\nidaqmx\system\_collections\physical_channel_collection.py
64:            return PhysicalChannel(self.channel_names[index])
66:            return PhysicalChannel(self.channel_names[index])
68:            return PhysicalChannel(f'{self._name}/{index}')
76:            yield PhysicalChannel(channel_name)
89:            yield PhysicalChannel(channel_name)
98:        return PhysicalChannel(flatten_channel_string(self.channel_names))

.\generated\nidaqmx\system\physical_channel.py
48:        return f'PhysicalChannel(name={self._name})'
```

Reply:
@bkeryan  Thanks for pointing this out. I replaced alternate constructor for functions in `system.py` and `physical_channel.py`. As this PR only deals with functions in autogenerated files, all other replacements will be handled in other PRs.